libvirt/tests/src/memory/memory_hotplug_virshdump.py

```
@@ -0,0 +1,289 @@
+            _vmxml = VMXML.new_from_inactive_dumpxml(vm_name)
+
+            _numa_present = False
+            try:
+                _cpuxml = _vmxml.cpu
+                if _cpuxml is not None and _cpuxml.numa_cell:
+                    _numa_present = True
+            except Exception:
+                pass
+
+            if not _numa_present:
+                logging.info("No NUMA nodes found on VM '%s' – configuring "
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)